### PR TITLE
Documentation about python debugging with VS Code

### PR DESCRIPTION
**Pull request by pfrydlewicz**

### CODE REVIEW

File: doc/manual/tutorial.rst [hunk 1]
```diff
@@ -306,3 +306,54 @@ For being able to announce the C++ filters, the plugin needs to be defined. This
     .. literalinclude:: ../../nexxT/tests/src/Plugins.cpp
         :language: c
 
+Debugging
++++++++++
+
```
Comment (cwiede) [inline doc/manual/tutorial.rst:311]: I would place a general comment here, that debugging can be achieved with any IDE which supports starting python modules (a.k.a. python -m <module>). The nexxT-gui start script can be replaced by python -m nexxT.core.AppConsole. Afterwards, the example with Visual Studio Code is ok.

File: doc/manual/conf.py [hunk 2]
```diff
@@ -73,8 +73,7 @@
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', 'manual']
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = None
```
[the remark this answers is not in the document]
Comment (pfrydlewicz) [reply doc/manual/conf.py:76]: I tried again, seems this is indeed not required.
This change can be dropped.

File: doc/manual/tutorial.rst (hunk 1 above)
Comment (pfrydlewicz) [reply doc/manual/tutorial.rst:312]: Already added mentioned changes.
Make sure to pull the HEAD of my fork.

**Comment by cwiede** [comment]
thanks for the PR - will be integrated with next release

**Pull request merged**